### PR TITLE
django-maintenance-mode: init at 0.13.1

### PR DESCRIPTION
**Pull request by MrMebelMan**
###### Motivation for this change

Adding a missing Django package.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/django-maintenance-mode/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchPypi, buildPythonPackage,
+  mock, django }:
+buildPythonPackage rec {
+  pname = "django-maintenance-mode";
+  version = "0.13.1";
```
Comment (lsix) [inline pkgs/development/python-modules/django-maintenance-mode/default.nix]: According to https://pypi.org/project/django-maintenance-mode/ there is a 0.13.3 version available. Any specific reason not to use it?

File: pkgs/development/python-modules/django-maintenance-mode/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "16yfw85r6my14vbrf9g2hj5p7ivgjvrd16s9prsjzqj9k8qikch2";
+  };
+
+  doCheck = false;
```
Comment (lsix) [inline pkgs/development/python-modules/django-maintenance-mode/default.nix]: Could you add a comment to explain why tests are disabled? (probably because setting up a django environment for nix-build can be problematic)

**Review by lsix — CHANGES_REQUESTED** [review]
Thanks for the PR,

I added comments in the review that should be addressed before merging.

File: pkgs/development/python-modules/django-maintenance-mode/default.nix (hunk 2 above)
Comment (risicle) [inline pkgs/development/python-modules/django-maintenance-mode/default.nix]: The tests just aren't included in the pypi package - they run fine if you fetch the source from github instead:

```
  src = fetchFromGitHub {
    owner = "fabiocaccamo";
    repo = pname;
    rev = version;
    sha256 = "06li0rbfc8f68y472dvp95sb5c9s5b15a4zjb2xdi5a7fndra4lr";
  };
```

Also note that the latest available version is 0.13.3

File: pkgs/development/python-modules/django-maintenance-mode/default.nix [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+  buildInputs = [ mock ];
```
Comment (risicle) [inline pkgs/development/python-modules/django-maintenance-mode/default.nix]: Why is this needed? It works (and the tests pass) for me without it (python 2.7 & 3.7).

File: pkgs/development/python-modules/django-maintenance-mode/default.nix (hunk 1 above)
Comment (MrMebelMan) [reply pkgs/development/python-modules/django-maintenance-mode/default.nix]: Well, not really, I am using it with an older version of django on one of my private projects... I'll bump it, sorry!

File: pkgs/development/python-modules/django-maintenance-mode/default.nix (hunk 2 above)
Comment (MrMebelMan) [reply pkgs/development/python-modules/django-maintenance-mode/default.nix]: Sure!

Comment (MrMebelMan) [reply pkgs/development/python-modules/django-maintenance-mode/default.nix]: Nice, I'll add them, thank you :pray:

File: pkgs/development/python-modules/django-maintenance-mode/default.nix (hunk 3 above)
Comment (MrMebelMan) [reply pkgs/development/python-modules/django-maintenance-mode/default.nix]: A good question indeed. I'll remove it and RTFM.

**Comment by MrMebelMan** [comment]
bumped the version to `0.13.3`, removed `mock`, changed `src` to fetch from github to be able to run tests, quoted the homepage URL

File: pkgs/development/python-modules/django-maintenance-mode/default.nix (hunk 1 above)
Comment (risicle) [reply pkgs/development/python-modules/django-maintenance-mode/default.nix]: I sympathize with being stuck on older versions of e.g. django - is dropping support for older djangos a deliberate move in 0.13.3 by the project authors? If not, do let them know it's not working for you.

File: pkgs/development/python-modules/django-maintenance-mode/default.nix [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  version = "0.13.3";
+
+  src = fetchFromGitHub {
+    owner = "fabiocaccamo";
+    repo = pname;
+    rev = version;
+    sha256 = "06li0rbfc8f68y472dvp95sb5c9s5b15a4zjb2xdi5a7fndra4lr";
+  };
+
+  buildInputs = [ pytest ];
```
Comment (FRidh) [inline pkgs/development/python-modules/django-maintenance-mode/default.nix]: `checkInputs`

Comment (MrMebelMan) [reply pkgs/development/python-modules/django-maintenance-mode/default.nix]: my bad :)

**Comment by mmahut** [comment]
@GrahamcOfBorg build django-maintenance-mode

**Comment by mmahut** [comment]
@MrMebelMan do you mind rebasing this?

**Comment by MrMebelMan** [comment]
> @MrMebelMan do you mind rebasing this?

done!

**Comment by mmahut** [comment]
@GrahamcOfBorg build django-maintenance-mode

**Comment by mmahut** [comment]
Sorry, dropped the ball on this, do you mind rebasing again?:)

**Comment by mmahut** [comment]
And maybe updating to latest.

**Comment by MrMebelMan** [comment]
Sorry for the late reaction. Rebased master & updated django-maintenance-mode to the latest version.

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 64520 [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>2 packages built:</summary>  <ul>    <li>python37Packages.django-maintenance-mode</li>    <li>python38Packages.django-maintenance-mode</li>  </ul></details>
